rdmo/core/tests/e2e/conftest.py

```diff
@@ -18,6 +18,10 @@ def _set_django_allow_async_unsafe():
     """pytest-playwright needs this setting to be enabled."""
     os.environ.setdefault("DJANGO_ALLOW_ASYNC_UNSAFE", "true")
 
+@pytest.fixture
```

Comment:
I think `settings.MULTISITE = True` is exactly as verbose as needed. Please remove this fixture. (Unless there is a reason I don't see, here.)

Reply:
yeah, I put it like that so that I could enable it only for that specific test under `management` but now the e2e `page` gets the multisite already, so then all the e2e tests are using it..
Otherwise I need to all of the setup of the page and login as well directly in the test..

Reply:
ok, so all e2e tests are multisite? fine with me.

Reply:
could also add another fixture `page_multisite` , to keep it separate